src/current/_includes/releases/v23.1/v23.1.24.md

```diff
@@ -0,0 +1,51 @@
+
+<h3 id="v23-1-24-{{-site.data.products.enterprise-}}-edition-changes">{{ site.data.products.enterprise }} edition changes</h3>
+
+- [`ALTER CHANGEFEED`]({% link v23.1/alter-changefeed.md %}) no longer removes the CDC query when modifying [changefeed]({% link v23.1/change-data-capture-overview.md %}) properties. [#125436][#125436]
+
+<h3 id="v23-1-24-operational-changes">Operational changes</h3>
+
+- Improved disk usage metric reporting over volumes that dynamically change their size over the life of the `cockroach` process. [#125106][#125106]
+- Removed `crdb_internal.cluster_execution_insights.txt` and `crdb_internal.cluster_txn_execution_insights.txt` have been removed from the [`debug zip`]({% link v23.1/cockroach-debug-zip.md %}). These files contained cluster-wide insights for statements and transactions. Users can still rely on the per-node execution insights in `crdb_internal.node_execution_insights.txt` and `crdb_internal.node_txn_execution_insights.txt`. [#125810][#125810]
+- Some debugging-only information about physcial plans is no longer collected in the `system.job_info` table for changefeeds, because it has the potential to grow very large. [#126102][#126102]
+
+<h3 id="v23-1-24-bug-fixes">Bug fixes</h3>
+
+- Fixed handling in the [declarative schema changer]({% link v23.1/online-schema-changes.md %}#declarative-schema-changer) when columns are included in the `STORING()` clause of [`CREATE INDEX`]({% link v23.1/create-index.md %}). CockroachDB now checks if the column is virtual up-front, and properly detects when a column is already handled by an existing index when the column name has `UTF-8` characters. [#125208][#125208]
+- Fixed a bug where a change to a [user-defined type]({% link v23.1/create-type.md %}) could cause queries against tables using that type to fail with an error message like: `histogram.go:694: span must be fully contained in the bucket`. The change to the user-defined type could come directly from an [`ALTER TYPE`]({% link v23.1/alter-type.md %}) statement, or indirectly from an [ALTER DATABASE ... ADD REGION]({% link v23.1/alter-database.md %}#add-region) or [`DROP REGION`]({% link v23.1/alter-database.md %}#drop-region) statement (which implicitly change the `crdb_internal_region type`). This bug had existed since UDTs were introduced in v20.2. [#124855][#124855]
+- Fixed an issue where [adding a column]({% link v23.1/alter-table.md %}#add-column) with a default value of an empty array would not succeed. [#125328][#125328]
+- [`ALTER TABLE ... ADD CONSTRAINT UNIQUE`]({% link v23.1/alter-table.md %}#add-constraint) will now fail with a well-formed error message and code `42601` if a statement tries to add a unique constraint on an expression. [#125420][#125420]
+- Fixed a bug in v24.1, v23.2, and v23.1 where using `changefeed.aggregator.flush_jitter` with `min_checkpoint_frequency` set to zero could cause panics. [#125495][#125495]
```

Comment:
maybe add a link on `changefeed.aggregator.flush_jitter` to  https://www.cockroachlabs.com/docs/v23.1/cluster-settings#setting-changefeed-aggregator-flush-jitter
maybe add a link on `min_checkpoint_frequency` to https://www.cockroachlabs.com/docs/v24.1/create-changefeed#min-checkpoint-frequency